common/djangoapps/student/tests/tests.py

```diff
@@ -234,7 +234,7 @@ def test_cert_grade(self, persisted_grade, cert_grade):
         Tests that the higher of the persisted grade and the grade
         from the certs table is used on the learner dashboard.
         """
-        expected_grade = max(persisted_grade, cert_grade)
+        expected_grade = max(filter(lambda x:x is not None, [persisted_grade, cert_grade]))
```

Comment:
Same here.

Reply:
@feanil from test data, we're certain about no instance with all `None` values, do we still want to make a change here